lib/content-services/src/lib/content-metadata/components/content-metadata-card/content-metadata-card.component.spec.ts

```diff
@@ -147,55 +139,6 @@ describe('ContentMetadataCardComponent', () => {
         expect(contentMetadataComponent).toBeNull();
     });
 
-    it('should toggle editable by clicking on the button', () => {
-        component.editable = true;
-        component.node.allowableOperations = [AllowableOperationsEnum.UPDATE];
-        fixture.detectChanges();
-
-        getToggleEditButton().triggerEventHandler('click', {});
```

Comment:
Hmm there are two other tests left which use getToggleEditButton function. That function refers to data-automation-id which is moved to different component. 
Are these 2 left unit tests still valid or should be removed/moved to component where that data-automation-id exists?

Reply:
@DenysVuika do we need to do something with this one?